index.html

```diff
@@ -27,13 +27,162 @@ <h2>I'm a Software Developer</h2>
         </div>
         <div class="social-media">
             <ul>
-                <li><a href="#"><img src="/images/github.png" alt="github"></a></li>
-                <li><a href="#"><img src="/images/linkedin.png" alt="linkedin"></a></li>
-                <li><a href="#"><img src="/images/anjelles.png" alt="anjelles"></a></li>
-                <li><a href="#"><img src="/images/twitter.png" alt="twitter"></a></li>
-                <li><a href="#"><img src="/images/medium.png" alt="medium icon"></a></li>
+                <li><a href="#"><img src="./images/github.png" alt="github"></a></li>
+                <li><a href="#"><img src="./images/linkedin.png" alt="linkedin"></a></li>
+                <li><a href="#"><img src="./images/anjelles.png" alt="anjelles"></a></li>
+                <li><a href="#"><img src="./images/twitter.png" alt="twitter"></a></li>
+                <li><a href="#"><img src="./images/medium.png" alt="medium icon"></a></li>
             </ul>
         </div>        
     </section>
+    <section class="works">
+        <h2 class="card-1">My Recent works</h2>
+        <div class="card-2">
+            <img src="./images/gym-image.png" alt="Placeholder">
+            <h3 id="story">Multi-Post Stories</h3>
+            <p>
+                A daily selection of privately personalized reads;
+                no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
+                when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="link-1 btn">
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">RUBY</a></li>
+            </ul>
+            <a class="see-project button-1" href="#">See project</a>
```

Comment:
- [ ] While you have done great progress, there are still some issues that are not fixed yet, 
- The button interactions 🔴 
- project images 🔴 

Please take a look at this record and see what to touch in order to make it perfect.

![5](https://user-images.githubusercontent.com/77843050/227721181-601c3de4-1593-47f6-865e-bca2b7466b2e.gif)

Reply:
Hello, the button interaction cant be seen on the mobile view because on mobile devices we don't have a cursor however on the desktop view we have a cursor hence interactions like hover can be seen.

Reply:
Hey @tumainimaganiko, don't worry about this, the next review will mark it as `solved`. I have checked again your code and it works fine.